packages/m/mavo.json

```
@@ -1,7 +1,7 @@
 {
   "name": "mavo",
   "description": "Create web applications with HTML and CSS",
-  "main": "mavo.js",
+  "filename": "mavo.js",
```

```suggestion
  "filename": "mavo.min.js",
```
Let's recommend the minified version while we're here